src/decorators/Mention/Suggestion/index.js

```diff
@@ -51,23 +51,34 @@ class Suggestion {
           0,
           selection.get('focusOffset') === text.length - 1 ? text.length : selection.get('focusOffset') + 1
         );
-        let index = text.lastIndexOf(separator + trigger);
-        let preText = separator + trigger;
-        if ((index === undefined || index < 0) && text[0] === trigger) {
-          index = 0;
-          preText = trigger;
+
+        const separatorList = [separator, '\n'];
+        let separatorIndex = -1;
+        let separatorWithTrigger = '';
+
+        separatorList.forEach(sep => {
+          const sepIndex = text.lastIndexOf(sep + trigger);
+          if (sepIndex > separatorIndex) {
+            separatorIndex = sepIndex;
+            separatorWithTrigger = sep + trigger;
+          }
+        });
```

Comment:
Please check this function for the case where both the separator come together for e.g (go to new line and then give space, and then press @) in this case it shows loading state and then closes.